src/hooks/useRepositoryStorage.js

```diff
@@ -19,6 +19,10 @@ const getStoredValue = async (key) => {
 function useRepositoryStorage() {
   const { data, mutate } = useSWR(ASYNC_STORAGE_KEY, getStoredValue);
 
+  const filteringArray = (id) => {
+    return data.filter((item) => item.id !== id);
+  };
```

Comment:
return이 한 줄이라면
```suggestion
  const filteringArray = (id) => data.filter((item) => item.id !== id);
```
으로 작성하시는 게 더 깔끔해보이네요!

Reply:
피드백 반영해서 수정해두었습니다!